composer.json

```diff
@@ -1,43 +1,51 @@
 {
-  "name": "ezsystems/ezplatform-search",
-  "description": "Platform Search bundle",
-  "type": "ezplatform-bundle",
-  "license": "GPL-2.0-only",
-  "autoload": {
-    "psr-4": {
-      "Ibexa\\Platform\\Bundle\\Search\\": "src/bundle/",
-      "Ibexa\\Platform\\Search\\": "src/lib/"
+    "name": "ibexa/search",
+    "description": "Ibexa DXP Search bundle",
+    "type": "ibexa-bundle",
+    "license": "GPL-2.0-only",
+    "replace": {
+        "ezsystems/ezplatform-search": "*"
+    },
+    "autoload": {
+        "psr-4": {
+            "Ibexa\\Platform\\Bundle\\Search\\": "src/bundle/",
+            "Ibexa\\Platform\\Search\\": "src/lib/",
+            "Ibexa\\Search\\": "src/lib/",
+            "Ibexa\\Bundle\\Search\\": "src/bundle/",
+            "Ibexa\\Contracts\\Search\\": "src/contracts/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Ibexa\\Platform\\Tests\\Bundle\\Search\\": "tests/bundle/",
+            "Ibexa\\Platform\\Tests\\Search\\": "tests/lib/",
+            "Ibexa\\Tests\\Search\\": "tests/lib/",
+            "Ibexa\\Tests\\Bundle\\Search\\": "tests/bundle/"
+        }
+    },
+    "require": {
+        "php": "^7.3",
+        "ezsystems/ezplatform-kernel": "^2.0@dev",
```

Comment:
```suggestion
        "ezsystems/ezplatform-kernel": "^4.0@dev",
```

Reply:
I missed it being merged already, I will update master and then rebase this one.